piker/data/_sampling.py

```diff
@@ -227,7 +227,8 @@ async def sample_and_broadcast(
             # end up triggering backpressure which which will
             # eventually block this producer end of the feed and
             # thus other consumers still attached.
-            subs = bus._subscribers[sym]
+            subs = bus._subscribers[sym.lower()]
```

Comment:
we're now internally normalizing all symbols to a lower case format for that *don't give af* style 😏